Formula/libffi.rb

```diff
@@ -17,24 +14,14 @@ class Libffi < Formula
   end
 
   head do
-    url "https://github.com/atgreen/libffi.git"
+    url "https://github.com/libffi/libffi.git"
     depends_on "autoconf" => :build
     depends_on "automake" => :build
     depends_on "libtool" => :build
   end
 
   keg_only :provided_by_macos
 
-  on_macos do
-    if Hardware::CPU.arm?
-      # Improved aarch64-apple-darwin support. See https://github.com/libffi/libffi/pull/565
-      patch do
-        url "https://raw.githubusercontent.com/Homebrew/formula-patches/06252df03c68aee70856e5842f85f20b259e5250/libffi/libffi-3.3-arm64.patch"
-        sha256 "9290aba7f3131ca19eb28fa7ded836b80f15cf633ffac95dc52b14d0a668d1fa"
-      end
-    end
-  end
```

Comment:
Note that the trampoline part of this patch has not been merged yet. Not sure if we have things using that or not.